api/scripts/helpers/csvHelpers.js

```diff
@@ -25,6 +25,15 @@ const optionsWithHeader = {
     if (columnName === 'createdBy') {
       value = !isEmpty(value) && parseInt(value, 10);
     }
+    if (columnName === 'credit' && isEmpty(value)) {
+      value = 0;
+    }
+    if (columnName === 'locale' && isEmpty(value)) {
+      value = 'fr-fr';
+    }
+    if (columnName === 'email' && !isEmpty(value)) {
+      value = value.replaceAll(' ', '').toLowerCase();
```

Comment:
Observations/Remarques : le replaceAll() remplace les espaces avant/après le mot mais également à l'intérieur, alors que le trim ne le fait qu'à l'extérieur, on n'avait effectivement pas besoin d'avoir les deux.
Par contre j'ai cherché la diff entre replace() et replaceAll() et à priori on utilise le all dans le cas d'ue regex, ce qui n'est pas le cas ici. On pourrait donc tout simplement mettre replace() ici 
```suggestion
      value = value.replace(' ', '').toLowerCase();
```

Reply:
il y a possiblement plus d'un espace et donc il faut remplacer tous pas juste le premier